docs/DafnyRef/UserGuide.md

```diff
@@ -932,6 +932,43 @@ method Slow(i: int, j: int)
 }
 ```
 
+Labelled assert statements are also available both in expressions and statements.
+Assertion labels are not accessible outside of scope of the assert statement.
```

Comment:
"the scope"

Might be better to say "the scope which the assert statement is in"